templates/cases/case_read.haml

```diff
@@ -161,11 +161,16 @@
             - trans "Contact"
           .panel-body
             .container-fluid
-              .row
+              .row{ ng-repeat:"urn in contact.urns" }
+                .contact-field-label.col-sm-6
+                  [[ urn.scheme ]]
+                .contact-field-value.col-sm-6
+                  [[ urn.path ]]
+              .row{ ng-hide:"contact.urns.length > 0" }
                 .contact-field-label.col-sm-6
                   - trans "Name"
                 .contact-field-value.col-sm-6
-                  [[ caseObj.contact.name ]]
+                  [[ contact.name ]]
```

Comment:
Ok so taking step back here, there are two things we want deployments to be able to customize:

1. What contact fields are allowed to be displayed
2. How a contact is displayed as single value

For our different requirements that means:

* For UPartners we want 1) to hide both URNS and name, and 2) to use an anon identifier based on UUID
* For Momconnect you need 1) to hide names and 2) the use a URN
* Another deployment might not have of our requirements and would 1) likely want to display all fields and 2) render contacts with their names - so this I think should be the default for CasePro.

To do that all of that I think we need to replace `SITE_ANON_CONTACTS` with something that handle 1), like `SITE_HIDE_CONTACT_FIELDS = ('name', ...)`. That setting determines what goes into the full JSON representation of a contact, and what we display on the contact read page. The name in the full representation should be the actual name unless `SITE_HIDE_CONTACT_FIELDS` contains name.

To make things simpler we can have an new field in the contact JSON representation called `display` whose content is determined by `SITE_CONTACT_DISPLAY`. For example the compact representation of a contact might look like:

```json
{"id": 1234, "display": "11EDA2"}  // UPartners
{"id": 1234, "display": "+25032432544"}  // Momconnect
{"id": 1234, "display": "Bob Jones"}  // Default
```

And the full representation...
```json
{"id": 1234, "display": "11EDA2", "name": null, "urns": [], ...}  // UPartners
{"id": 1234, "display": "+25032432544", "name": null, "urns": ["tel:+25032432544"], ...}  // Momconnect
{"id": 1234, "display": "Bob Jones", "name": "Bob Jones", "urns": ["tel:+25032432544"], ...}  // Default
```

I think the separate `display` field keeps things simpler and the introduction of `SITE_HIDE_CONTACT_FIELDS` will be helpful when we get to the point of wanting to edit contacts etc. Thoughts?